docs/learn-parachains.md

```diff
@@ -53,6 +53,30 @@ could not include that in their implementation and Polkadot would still enforce
 Parachains are not required to have their own token. If they do, is up to the parachain to make the
 economic case for their token, not Polkadot.
 
+## Parachain Hubs
+
+While Polkadot enables crosschain functionality amongst the parachains, it necessitates that there
+is some latency between the dispatch of a message from one parachain until the desitnation parachain
+recieves the message. In the optimistic scenario, the latency for this message should be at least 2
+blocks, one block for the message to be dispatched and one block for the receiving parachain to
+process and produce a block that acts upon the message. However, in some cases we may see that the
+latency for messages are higher if there is a lot of messages that are in queue to be processed, or
+if there exists no nodes that are running both of the parachain networks that can quickly gossip the
+message across the networks.
+
+Due to the neccesary latency involved in sending crosschain messages, some parachains are planning
+to become _hubs_ for an entire industry. For example, a parachain project
+[Acala](https://acala.network) is planning to become a hub for decentralized finance (DeFi)
```

Comment:
How does this work, concretely? It's assumed there would still be DeFi products on other chains, like Moonbeam. How does Acala help with this? Or is the idea behind a hub simply that they plan to entice all DeFi projects to just build on them?

Reply:
The idea of the hub is primarily that it makes composability between applications easier. That is, both Acala and Moonbeam will have smooth interoperability between applications on their own parachain. While applications on Moonbeam would experience latency when interacting with anything that's deployed to Acala and vice versa.

Reply:
So literally "they plan to do what eth is doing now and concentrate all apps and protocols on them", so it's a battle for dev attention between the chains?